modules/instruments/test/TestPlan.md

```diff
@@ -0,0 +1,18 @@
+# Instrument Test Plan
+
+1. Verify that in order to access the module, the user must either have the `Across all sites access
+candidate profiles` permission, or have the 'Edit Candidate Parameters' permission and be at the same
+site as the visit, or at the same site as one of the candidate's other visits.
+2. Select and instrument in the list of the session instruments. Note that the visit's stage should
```

Comment:
the stage does not necessarily need to be at visit, you can remove that section. regardless of the stage instruments can be populated depending on the test_battery assignments